### PR TITLE
Chang's post on using AngularJS components in React

### PR DESCRIPTION
**Pull request by cheapsteak**

### CODE REVIEW

**Comment by andricDu** [comment]
:+1: for Oxford comma

File: _posts/chang_wang/2018-01-01-Using-AngularJS-components-directives-in-React.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      ref={c => { this.container = c; }}
+      dangerouslySetInnerHTML={{ __html: '<widget></widget>' }}
+    />
+  )
+}
+```
+
+## Summary  
+
+It's always a difficult decision on whether, when, and how to migrate from a framework. For those that have started or are about to begin, I hope this is article will help ease and complete that transition.
```
Comment (Jephuff) [inline _posts/chang_wang/2018-01-01-Using-AngularJS-components-directives-in-React.md]: `I hope this is article` extra "is" in there I think

Comment (cheapsteak) [reply _posts/chang_wang/2018-01-01-Using-AngularJS-components-directives-in-React.md]: thanks!

File: _posts/chang_wang/2018-01-01-Using-AngularJS-components-directives-in-React.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+If the old routes and new routes don't match 1:1, you could insert a mapping function that maps old routes to new routes. An example of that can be found in our codebase [here](https://github.com/NCI-GDC/portal-ui/blob/7dd05963c638d1ff9ae60b4f2eb089151aa1a473/modules/node_modules/%40ncigdc/components/GitHut.js#L86-L110)
+
+## Neutralize Angular's location tampering  
+
+A problem we ran into was that after initializing and then navigating away from the Angular component, the browser's address bar would become "stuck", always resetting to the route the angular component was on. 
+
+Keep an eye on the address bar:
+
+<video class="share-video" id="share-video" poster="https://thumbs.gfycat.com/SizzlingKeyDolphin-poster.jpg" autoplay="" muted="" loop>
```
Comment (Fgerthoffert) [inline _posts/chang_wang/2018-01-01-Using-AngularJS-components-directives-in-React.md]: Should you host those files on github pages (i.e. the softeng-blog repo) ?

**Review by Fgerthoffert — APPROVED** [review]
Well done Chang, very good article !

My only comment is more about not having too many dependencies to external services (i.e. video or code hosting) in the article's content (I'm not referring to external links). 
But this might not be a valid point, so I let you decide what you find best.

File: _posts/chang_wang/2018-01-01-Using-AngularJS-components-directives-in-React.md (hunk 2 above)
Comment (cheapsteak) [reply _posts/chang_wang/2018-01-01-Using-AngularJS-components-directives-in-React.md]: I can do that :D

**Comment by cheapsteak** [comment]
Moved dependencies inside  :D

**Pull request merged**